substrate/frame/asset-conversion/src/lib.rs

```diff
@@ -413,6 +413,11 @@ pub mod pallet {
 		/// Params `amount1_min`/`amount2_min` represent that.
 		/// `mint_to` will be sent the liquidity tokens that represent this share of the pool.
 		///
+		/// NOTE: when encountering an incorrect exchange rate and non-withdrawable pool liquidity,
```

Comment:
This is totally subjective but it might be better to say:
```suggestion
		/// NOTE: to rectify an incorrect exchange rate and non-withdrawable pool liquidity,
```

Which I think makes it a bit clearer.

Reply:
Apologies, non-withdrawable pool liquidity can't be rectified as you pointed out. Wording is fine as is